.github/ISSUE_TEMPLATE/bug-issue.yml

```diff
@@ -0,0 +1,72 @@
+name: Bug report
+description: Report a very clearly broken issue.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # osu! bug report
+
+        Important to note that your issue may have already been reported before. Please check:
+        - Pinned issues, at the top of https://github.com/ppy/osu/issues.
+        - Current open `priority:0` issues, filterable [here](https://github.com/ppy/osu/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Apriority%3A0).
+        - And most importantly, search for your issue. If you find that it already exists, respond with a reaction or add any further information that may be helpful.
+
+  - type: dropdown
+    attributes:
+      label: Type
+      options:
+        - Crash to desktop
+        - Game behaviour
+        - Performance
+        - Cosmetic
+        - Other
```

Comment:
I'm not sure these types are doing much. What was the intention with these? I'm not sure I'd be able to always fill this out correctly, let alone users.

Reply:
This was brought on request by @peppy, and I believe it is not that hard to fill:
 - First is for crash reports.
 - Second is for general bug reports.
 - Third is for performance bug reports.
 - Fourth is for UI error reports (misalignment, text not wrapping, etc.)

We can consider grouping "Cosmetic" and "Game behaviour" under one option and call it a day however (as also mentioned by @peppy in discord, which I'm fine with if that's the issue here).

Reply:
Intention was to potentially map to labels down the track, and give the user a bit of prodding in direction - making sure they know what they are reporting at least at a high level. I'd like to give it a try but if it's deemed non-useful can be removed.